pants

```diff
@@ -62,8 +62,25 @@ function get_pants_ini_config_value {
   sed -ne "/${prefix}/ s#${prefix}##p" pants.ini
 }
 
+function get_python_major_minor_version {
+  python_exe="$1"
+  "$python_exe" <<EOF
+import sys
+major_minor_version = ''.join(str(version_num) for version_num in sys.version_info[0:2])
+print(major_minor_version)
+EOF
+}
+
+function check_interpreter_on_path {
+  python="$1"
+  if ! which "${python}" >/dev/null; then
+    die "Could not find ${python_bin_name}. Please ensure ${python_bin_name} is on your PATH."
```

Comment:
It's implicit that `python_bin_name` will be set by this point... would probably be preferable to use the parameter or explicitly pass a human readable form if `$1` isn't readable.

Reply:
This was 100% a bad merge! 

We intentionally want to keep this function generic, because in the future when we (maybe) auto-generate `pants.ini` I'm envisioning our flow will look something like this: `try py36 -> try py27 -> try py37 -> fail`.

This function—with modifications—will be the basis for that flow.

Thanks!